src/tests/ftest/io/unaligned_io.yaml

```diff
@@ -25,5 +25,3 @@ datasize:
     - 300
     - 16000
     - 1048599
-    # Skip this large size until DAOS-2164 resolved.
-    #- 1073741824
```

Comment:
Are you not planning to use cancelForTicket? I thought we should start using it...

Reply:
This is a large transfer size and it's not supported by SPDK so removing the tests.

Reply:
ok.